OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -18,15 +19,16 @@ void TracepointEventBuffer::AddTracepointEventAndMapToThreads(uint64_t time,
   event.set_tid(thread_id);
   event_map[time] = std::move(event);
 
-  // Add all tracepoint events to "all threads".
-  std::map<uint64_t, orbit_client_protos::TracepointEventInfo>& event_map_all_threads =
-      tracepoint_events_[SamplingProfiler::kAllThreadsFakeTid];
-  orbit_client_protos::TracepointEventInfo event_all_threads;
-  event_all_threads.set_time(time);
-  event_all_threads.set_tracepoint_info_key(tracepoint_hash);
-  event_all_threads.set_tid(SamplingProfiler::kAllThreadsFakeTid);
-  event_map_all_threads[time] = std::move(event);
-
+  if (is_same_pid_as_target) {
+    // Add all tracepoint events to "all threads".
```

Comment:
This comment is now outdated as it's not "all tracepoint events" but only the ones related to the target process.

Reply:
Of course. Thank you!